apps/src/music/player/MusicPlayer.ts

```diff
@@ -297,7 +281,7 @@ export default class MusicPlayer {
   // Converts actual seconds used by the audio system into a playhead
   // position, which is 1-based and scaled to measures.
   convertSecondsToPlayheadPosition(seconds: number): number {
-    return 1 + seconds / this.secondsPerMeasure();
+    return Math.round((1 + seconds / this.secondsPerMeasure()) * 1000) / 1000;
```

Comment:
Curious: why this change?

Reply:
ah good catch, was leftover from a different approach where I was using the playhead time as an event identifier. though, maybe it's worth keeping this around if we want to keep playhead values rounded? I was seeing values like 1.9999999998

Reply:
Discussed offline - removing rounding to preserve precision.